frontend/src/app/shared/components/file-links/file-picker-modal/file-picker-modal.html

```diff
@@ -0,0 +1,57 @@
+<div
+  class="spot-modal spot-modal_wide"
+>
+  <div class="spot-modal--header">
+    <span class="spot-modal--header_title">{{text.header}}</span>
+    <button
+      class="button button_no-margin"
+      (click)="openStorageLocation()"
+    >
+      <span>{{text.buttons.openStorage()}}</span>
+      <span class="spot-icon spot-icon_external-link"></span>
+    </button>
+  </div>
+
+  <div class="spot-divider"></div>
+
+  <div class="spot-modal--body spot-container">
+    <ng-container *ngIf="(loading$ | async) === false; else loadingTemplate">
+      <ul
+        class="spot-list spot-list_compact op-files-tab--file-list"
```

Comment:
Using `op-files-tab` elements here is wrong in two ways:

1. We're missing the `op-files-tab` block itself,
2. We're not in the files tab here, so the naming makes no sense.

If you need these styles to be available in both place I would recommend extracting out the `op-files-tab--file-list` element into its own block (e.g. `op-files-list`) and then using this new block.

Reply:
I extracted `op-file-list` as a separate block. Good idea.